libs/isograph-swc-plugin/.gitignore

```diff
@@ -0,0 +1,3 @@
+.swc
+node_modules
```

Comment:
let's remove this and add `**/swc_isograph_plugin.wasm` to the root `.gitignore`